### ISSUE TITLE
See history of campaigns sent to a subscriber

### ISSUE THREAD

**Issue by MaximilianKohler** [knadh/listmonk#1198]
Currently when you click on a subscriber it only shows what lists they're in and their bounce history (if exists). I think it would be nice for it to also show which campaigns they were sent. 

In order for me to keep track of that right now I have to create a new list for each campaign.

**Comment by knadh** [comment]
I'd thought about this. To record this per subscriber in a database of millions of subscribers is to insert millions of rows on every campaign which can quickly overload the database.

**Comment by MaximilianKohler** [comment]
Does using lists as an alternative also overload it?

**Comment by knadh** [comment]
Every subscriber-list connection adds a new entry in the database in the `subscriber_lists` table. This is unavoidable.

**Comment by MaximilianKohler** [comment]
I think that was a "yes"? I'm also overloading the DB by using lists like this? 

Is it just that I'll need to pay more for larger storage, or I'll need to pay for more compute power as well?

**Comment by knadh** [comment]
This is an issue if you've a large number of subscribers running into the millions. For tens of thousands, it shouldn't be a problem. listmonk requires very little CPU/RAM to operate on DBs with millions of subscribers. Don't think that would be an issue.

**Comment by MaximilianKohler** [comment]
Yeah I understood that. But I may very well get up to millions, so I'm trying to understand the exact impact of these decisions.
